fibonaci/fibonacci.py

```diff
@@ -0,0 +1,122 @@
+from decimal import Decimal, getcontext
+import math
+from functools import lru_cache
+
+__all__ = [
+    "fibonacci_closed",
+    "fibonacci_loop",
+    "fibonacci_range",
+    "fibonaci_loop_max",
+    "fibonacci_recursive",
+]
+
+
+def fibonacci_closed(n: int) -> int:
+    """Function that returns the nth number in the fibonacci sequence
+
+    Args:
+        n (int): Return the number of the fibonacci sequence of this index
```

Comment:
Zie er netjes uit, jongens.

Aangezien je in de functie declaratie al hebt aangegeven wat de types zijn, hoef je dat hier niet nog een keer te doen.